eva-server/src/main/java/uk/ac/ebi/eva/server/repository/VariantEntityRepositoryCustom.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2015 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.server.repository;
+
+import uk.ac.ebi.eva.commons.models.metadata.VariantEntity;
+
+import java.util.List;
+
+/**
+ * Interface to declare custom repository methods.
```

Comment:
Since a Spring Data repository interface can contain extra methods, maybe this could clarify it a bit more?

> Interface to declare additional repository methods with a custom implementation, instead of the one that Spring Data would provide by default.